src/main/java/de/baleipzig/iris/ui/view/recognition/RecognitionView.java

```diff
@@ -21,14 +23,19 @@
 
     private final ApplicationContext context;
 
+    @Getter
```

Comment:
kann man das in ne BaseView packen und von der erben? weil wird auch in der TrainingView